pkg/handler/features_test.go

```diff
@@ -63,6 +63,9 @@ type FeatureTestCase struct {
 
 func TestFeatures(t *testing.T) {
 	config.Get().Features.Snapshots.Enabled = true
+	config.Get().Features.AdminTasks.Enabled = true
+	defer resetFeatures()
```

Comment:
I had to make this change to get this test to pass. It seems like something that should be failing on every PR, but it only seems to fail on mine. I can't figure out why.

Reply:
I made similar changes on other tests.

Reply:
Locally, if I checkout main, and disabled admin_tasks in my config, this test fails.

Reply:
so with these changes, this test isn't failing for you anymore if you disable admin_tasks?

Reply:
that's strange. i see the failure on the TestCreate repository handler test when removing the lines you added there. i'm not sure why either, @jlsherrill do you have an idea?